docs/source/install.rst

```diff
@@ -12,15 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using `conda <https://www.anaconda.com/download/>`_::
+You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
 
-    conda install dask
+   ``conda install dask``
 
-This installs Dask and all common dependencies, including Pandas and NumPy.
+This installs Dask and **all** common dependencies, including Pandas and NumPy.
 
-Dask packages are maintained both on the default channel and on and
-`conda-forge <https://conda-forge.github.io/>`_.
+Dask packages are maintained both on the default channel and on `conda-forge <https://conda-forge.github.io/>`_.
 
+Optionally, you can obtain a minimal dask installation using:
```

Comment:
Perhaps "using the following command::"

Reply:
Done, plus I've taken out an extra ":" that snuck in somewhere else.

Reply:
The extra `:` is intentional.  http://docutils.sourceforge.net/docs/user/rst/quickref.html#literal-blocks

Reply:
See my new comment below, it's not that colon 😳 But thanks for the rst link, I'll definitely need to practice that more 😃